File: testes/testComp/3.casos_teste_semantico_gerador/saida/valido_typescript/controllers/users.controller.ts
import { Controller, Get, Post } from '@nestjs/common';
import { UserDto } from '../dtos/user.dto';

@Controller()
export class UsersController {

  @Get('/users')
  getUsers(): UserDto[] {
    throw new Error('Not implemented');
  }

  @Post('/users')
  createUser(): UserDto {
    throw new Error('Not implemented');
  }
}

File: testes/testComp/3.casos_teste_semantico_gerador/saida/valido_kotlin/controllers/UsersController.kt
package com.example.generated.controllers

import com.example.generated.dtos.UserDto
import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping
class UsersController {

    @GetMapping("/users")
    fun getUsers(): List<UserDto> {
        TODO("Not yet implemented")
    }

    @PostMapping("/users")
    fun createUser(): UserDto {
        TODO("Not yet implemented")
    }
}
